app/models/agents/user_location_agent.rb

```diff
@@ -68,7 +73,16 @@ def receive_web_request(params, method, format)
     def handle_payload(payload)
       location = Location.new(payload)
 
-      if location.present?
+      if interpolated[:accuracy_field].present?
+        accuracy_field = interpolated[:accuracy_field]
+      else
+        accuracy_field = 'accuracy'
+      end
+
+      if location.present? && (!interpolated[:max_accuracy].present? || !payload[accuracy_field] || payload[accuracy_field].to_i < interpolated[:max_accuracy].to_i)
+        if !payload[accuracy_field]
```

Comment:
Don't log unless the user cares: `if interpolated[:max_accuracy].present? && !payload[accuracy_field].present?`